arwen/contexts/async.go

```diff
@@ -815,6 +558,9 @@ func (context *asyncContext) DetermineExecutionMode(destination []byte, data []b
 	return arwen.AsyncUnknown, nil
 }
 
+// TODO decide whether this function is necessary at all, because unspent gas should
+// be accummulated into the AsyncContext, then refunded to the original caller.
+// Redistribution of gas among calls in the same group should not be necessary.
 func (context *asyncContext) setupAsyncCallsGas() error {
```

Comment:
please delete redistribution in the next PR.

Reply:
Will do.